cpp/CMakeLists.txt

```diff
@@ -79,37 +143,24 @@ add_library(NVText SHARED
             src/custring_view.cu
             src/custring.cu)
 
+###################################################################################################
+# - link libraries --------------------------------------------------------------------------------
+
 target_link_libraries(NVStrings rmm)
 target_link_libraries(NVCategory NVStrings rmm)
 target_link_libraries(NVText NVStrings rmm)
 
-add_library(pyniNVStrings SHARED python/pystrings.cpp)
-add_library(pyniNVCategory SHARED python/pycategory.cpp)
-add_library(pyniNVText SHARED python/pytext.cpp)
-
-if (WIN32) 
-set_target_properties(pyniNVStrings PROPERTIES SUFFIX ".pyd")
-set_target_properties(pyniNVCategory PROPERTIES SUFFIX ".pyd")
-set_target_properties(pyniNVText PROPERTIES SUFFIX ".pyd")
```

Comment:
Just out-of-curiosity, were we building Windows packages before?

Reply:
I believe one of the developers develops on Windows and they added this line, but otherwise we are not intending on releasing or supporting Windows packages at this time.